.rubocop.yml

```diff
@@ -54,6 +61,10 @@ Layout/MultilineOperationIndentation:
 Layout/ParameterAlignment:
   EnforcedStyle: with_first_parameter
 
+Layout/SpaceInsideHashLiteralBraces:
+  EnforcedStyle: no_space
+  EnforcedStyleForEmptyBraces: space
```

Comment:
I'm not sure we want this, how many are empty with a space before this change vs the ones changed?

Reply:
I'd go with `no_space`/`no_space` if possible.

Reply:
That would be my preference, but only if it matches what we mostly do.

Reply:
EnforcedStyle: no_space
  EnforcedStyleForEmptyBraces: space
73 offences

  EnforcedStyle: space
  EnforcedStyleForEmptyBraces: space
111 offences

  EnforcedStyle: space
  EnforcedStyleForEmptyBraces: no_space
73 offences

  EnforcedStyle: no_space
  EnforcedStyleForEmptyBraces: no_space
37 offences

Added a commit to fix that.